kalastatic.module

```diff
@@ -73,3 +73,93 @@ function kalastatic_preprocess_html(&$variables) {
     $variables['build_path'] = base_path() . 'kalastatic/';
   }
 }
+
+/**
+* Prepare the variables from a text_with_summary field.
+ */
+function kalastatic_prepare_field__text_with_summary($field, $entity) {
+  $output = [];
+  foreach ($entity->$field->getValue() as $i => $item) {
```

Comment:
text.value/text.summary is a good idea but it breaks our pre-established convention with text though. I guess we may end up with some extra layers in our json data on the prototype side but it's probably worth it to ensure compatibility with the Derpals